server/index.js

```diff
@@ -50,6 +76,33 @@ app.get('/api/accounts/:username', (req, res, next) => {
     .catch(err => next(err));
 });
 
+// ---------------------------- PUT (UPDATE) REQUESTS ---------------------//
+
+app.put('/api/accounts/new-rating/:username&:newRating', (req, res, next) => {
+  const sql = `
+    UPDATE "accounts"
+    SET "currentRating" = $2
+    WHERE "username" = $1
+    returning *
+  `;
+  const username = req.params.username;
+  const newRating = req.params.newRating;
+  const params = [username, newRating];
+  db.query(sql, params)
+    .then(result => {
+      if (!result.rows[0]) {
+        res.status(404).json({
+          error: 'No users found'
+        });
+      } else {
+        res.json(result.rows[0]);
+      }
+    })
+    .catch(err => next(err));
+});
```

Comment:
We don't really need to store the "current" rating in a separate route, we can do one of two things:

1. remove the `currentRating` entirely, and just calculate it in the SQL query that retrieves the average rating by joining the ratings table 
2. or, if there's concern that this is too expensive, we can update the `currentRating` when the user gets rated. Either way, I don't think another route is necessary.